win32ss/gdi/ntgdi/init.c

```diff
@@ -7,10 +7,13 @@
  */
 
 #include <win32k.h>
+DBG_DEFAULT_CHANNEL(UserMisc);
 
-#define NDEBUG
-#include <debug.h>
-#include <kdros.h>
+//#define NDEBUG
+//#include <debug.h>
+//#include <kdros.h>
```

Comment:
It looks like those things are unneeded in this file. If compilation succeeds without these, I'll clean them up.